.ci/scripts/upper_bound.py

```diff
@@ -3,10 +3,11 @@
 packages = []
 with open("requirements.txt", "r") as fd:
     for line in fd.readlines():
-        req = Requirement.parse(line)
-        spec = str(req.specs)
-        if len(req.specs) < 2 and "~=" not in spec and "==" not in spec and "<" not in spec:
-            packages.append(req.name)
+        if not line.startswith("#"):
```

Comment:
could you add it on plugin-template?

Reply:
i was planning too, i just used this PR as guinea pig ;)